solution/backend/regcore/migrations/0012_auto_20220923_1156.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 3.2.13 on 2022-09-23 11:56
+
+from django.db import migrations, models
+import django
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('regcore', '0011_parserconfiguration_retries'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='parserconfiguration',
+            name='skip_versions',
+        ),
+        migrations.AddField(
+            model_name='parserconfiguration',
+            name='skip_fr_documents',
+            field=models.BooleanField(default=True, help_text='Should the Federal Register parser skip processing documents that have been previously processed?'),
```

Comment:
would it be easier to read if we don't have all these in one long line?

Reply:
Migrations are automatically generated and we try not to edit them unless writing a custom one